src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.xml

```diff
@@ -20,6 +20,7 @@
         <!-- Normalizer -->
         <service id="serializer.normalizer.object" class="Symfony\Component\Serializer\Normalizer\ObjectNormalizer" public="false">
             <argument type="service" id="serializer.mapping.class_metadata_factory" />
+            <argument>null</argument> <!-- Replaced by the extension if a name converted is configured -->
```

Comment:
The comment should only be about what this is about, so something like `<!-- name converted -->`